SnapSpire/Features/Discover/View/SectionListView.swift

```diff
@@ -0,0 +1,33 @@
+import SwiftUI
+
+protocol SectionListViewMoldeProtocol: ObservableObject {
+    var topics: [String] { get }
+}
+
+struct SectionListView<Model>: View where Model: SectionListViewMoldeProtocol {
```

Comment:
Why do you need a protocol here? Also there is a typo in the name

Reply:
I don't need it, I have also restructured this part.